### PR TITLE
docs(asset-time-series-panel): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-540.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTimeseriesPanel/stories/examples.story.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<Preview withSource='open'>
+  <Story name="Custom Styles">  
+    <AssetTimeseriesPanel
+      assetId={4650652196144007}
+      styles={customStyle}
+    />
+  </Story>
+</Preview>
+
+## Custom Time Series Chart Meta
```
Comment (polomani) [inline src/components/AssetTimeseriesPanel/stories/examples.story.mdx]:
```suggestion
## Custom TimeSeriesChartMeta
```

File: src/components/AssetTimeseriesPanel/stories/basic.story.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
+import { AssetTimeseriesPanel } from '../AssetTimeseriesPanel';
+import { decorators } from './helper';
+
+<Meta title="Assets|AssetTimeseriesPanel" decorators={decorators} />
+
+# Asset Timeseries Panel
+
+#### Description:
```
Comment (polomani) [inline src/components/AssetTimeseriesPanel/stories/basic.story.mdx]:
```suggestion
#### Description:
```

File: src/components/AssetTimeseriesPanel/stories/examples.story.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { decorators, customStyle, metaOptions, handleAssetTimeseriesLoaded } from './helper';
+import { AssetTimeseriesPanel } from '../AssetTimeseriesPanel';
+
+<Meta title="Assets|AssetTimeseriesPanel/Example" decorators={decorators} />
+
+## Custom Spinner
+      
+<Preview withSource='open'>
```
Comment (polomani) [inline src/components/AssetTimeseriesPanel/stories/examples.story.mdx]: @mrtraser can we define this somewhere in one place?

Comment (mrtraser) [reply src/components/AssetTimeseriesPanel/stories/examples.story.mdx]: I've just updated `docs/configure-addon-docs` branch with set code examples expanded by default

Comment (mrtraser) [reply src/components/AssetTimeseriesPanel/stories/examples.story.mdx]: so here is no need to define `withSource='open'` here

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-540.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTimeseriesPanel/stories/basic.story.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
```
Comment (mrtraser) [inline src/components/AssetTimeseriesPanel/stories/basic.story.mdx]: we don't need `Source` import here.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-540.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTimeseriesPanel/stories/helper.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  MockDatapointsClientObject,
+  MockTimeseriesClientObject,
+  TimeseriesMockClient,
+} from '../../TimeseriesChart/stories/TimeseriesChart.stories';
+
+class CogniteClient extends TimeseriesMockClient {
+  timeseries: any = {
+    ...MockTimeseriesClientObject,
+    list: (): CogniteAsyncIterator<GetTimeSeriesMetadataDTO[]> => ({
+      // @ts-ignore TODO - remove this ts-ignore after fixing SDK
```
Comment (mrtraser) [inline src/components/AssetTimeseriesPanel/stories/helper.tsx]: think this comment can be removed

Comment (amevck) [reply src/components/AssetTimeseriesPanel/stories/helper.tsx]: Still, an error shows if remove this comment

Comment (mrtraser) [reply src/components/AssetTimeseriesPanel/stories/helper.tsx]: strange
ok, leave it as is for now

**Pull request merged**